### PR TITLE
Adds a class instance to Data.Csv.ToField for Month type

### PR DESCRIPTION
**Pull request by jarcane**
This lets us use month as a value for the dynamic report tables.

### CODE REVIEW

File: src/Futurice/Time/Month.hs [hunk 1]
```diff
@@ -167,6 +168,9 @@ instance ToHtml Month where
     toHtmlRaw = toHtml
     toHtml = toHtml . monthToText
 
+instance Csv.ToField Month where
```
Comment (phadej) [inline src/Futurice/Time/Month.hs]: you should add `Csv.FromField` at the same time :)

Comment (jarcane) [reply src/Futurice/Time/Month.hs]: Good point, though that might be harder. :D

File: src/Futurice/Time/Month.hs [hunk 2]
```diff
@@ -112,6 +113,9 @@ instance Show Month where
 -- TODO write Read instance to match above Show instance
 
 instance Hashable Month
+instance Csv.ToField Month
```
Comment (neurogoo) [inline src/Futurice/Time/Month.hs:116]: How does the generic version look? Is the generic version output same as Csv.toField . show?

Comment (jarcane) [reply src/Futurice/Time/Month.hs:116]: It appears to be exactly identical to the version I did manually.

**Review by neurogoo — APPROVED** [review]

**Comment by phadej** [comment]
```haskell
src/Futurice/Time/Month.hs:116:10: error: [-Wmissing-methods, -Werror=missing-methods]
    • No explicit implementation for
        ‘Csv.toField’
```

There is no generics implementations for `Csv.ToField` and `Csv.FromField`.

(EDIT: in fact those warnings are visible in travis, but they are warnings, not errors).

**Pull request merged**